.github/workflows/ci.yml

```diff
@@ -34,21 +34,12 @@ jobs:
           submodules: true
 
       - name: Use OCaml ${{matrix.ocaml_compiler}}
-        if: runner.name != 'macos-arm'
         uses: ocaml/setup-ocaml@v2
```

Comment:
Curious this does not need to change. Not very transparent.
It means things *will* change behind your back in future.